pyflakes/test/test_type_annotations.py

```diff
@@ -507,6 +507,51 @@ def f(x: Literal['some string']) -> None:
             return None
         """)
 
+    @skipIf(version_info < (3,), 'new in Python 3')
+    def test_forward_ref_valid(self):
+        self.flakes("""
+        from typing import Optional
+
+        def f(x: Optional['int']) -> None:
+            return None
+        """)
+
+    @skipIf(version_info < (3,), 'new in Python 3')
+    def test_forward_ref_missing(self):
+        self.flakes("""
+        from typing import Optional
+
+        def f(x: Optional['integer']) -> None:
+            return None
+        """, m.UndefinedName)
```

Comment:
these two tests are already tested above, I don't think this adds any coverage that isn't already there

Reply:
Thanks, I removed those tests.